### PR TITLE
fix: An Ungodly Ammount of Typos

### PR DESCRIPTION
**Pull request by KieranRobson**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [x] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

---

Fixing some typos here, alot are the same fixes but no rush to merge as there are alot here!

### CODE REVIEW

**Comment by dougaitken** [comment]
```
Ammount
```

I see what you did there 😏

**Comment by KieranRobson** [comment]
> ```
> Ammount
> ```
> 
> I see what you did there 😏

That was totally deliberate... yes 👀

**Comment by dougaitken** [comment]
This will take some time. I've found out. There is a difference between cowork and co-work so some instances might make sense. 

Also e-commerce vs ecommerce is a tricky one as well. 

Maybe I should get a glossary on this.

**Comment by KieranRobson** [comment]
Good plan, I'll look into and could help, maybe pop it in the pr template like

- Any technical terms such as cowork or ecommerce match the [glossary definition] (page link)

**Comment by KieranRobson** [comment]
I use a spell checker and it didn't like coworking so I updated as I went through

I've now read up on the differences so will go back through and update!

**Comment by dougaitken** [comment]
Right, I was reading up a bit on this, one seems to be aimed at where you literally work in a place with others, and the other is when you work with your colleagues. 

It will take a bit to go through all the instances of this so I'm not sure how to draw this one.

**Comment by KieranRobson** [comment]
We could close it and ignore all instances of cowork and hope theyre correct 😆 😉

**Comment by dougaitken** [comment]
There's still a bunch in here I want though haha!

**Comment by KieranRobson** [comment]
Will get this sorted tomorrow so we can get it merged! Been a bit busy with other projects and PhD to sit down and work through it

**Comment by KieranRobson** [comment]
@dougaitken All co(-)working changes have been reverted other than the typod one!

File: company-profiles/intellum.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ USA - Atlanta, Georgia.
 
 ## How to apply
 
-You can see a listing of opportunities here https://apply.workable.com/intellum-inc/ and on Intellum's LinkedIn profile.
-Intellum's website does not seem to have a career oppotunity section.
+You can see a listing of opportunities here https://apply.workable.com/intellum-inc/ and on Intellum's LinkedIn profile.oppu
```
Comment (dougaitken) [inline company-profiles/intellum.md:37]: the `oppu` at the end, typo?

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by dougaitken** [comment]
A single typo left @KieranRobson

**Comment by KieranRobson** [comment]
*now* no more typos, although I haven't went through the newly merged ones, may have missed a typo or two

**Review by dougaitken — APPROVED** [review]

**Pull request merged**